### DISCUSSION TITLE
How to catch the Errors thrown by Larascord?

### DISCUSSION THREAD

**Discussion by thephilluk** [JakyeRU/Larascord#64] [Q&A] [answered]
Hi there,
I am currently working on a little site with role only login.
How can I "catch" the Error messages from Larascord?

right now I am thrown to a json view of the data in the larascord Config: 
![image](https://github.com/JakyeRU/Larascord/assets/36800842/ffd5f83b-0418-4e54-8c6f-a4e773819de9)

Any help is appreciated!
Philip

**Comment by ValentinNeaga** [comment, marked as answer]
Hello @thephilluk,

That JSON response is meant for debugging. It will only show if `APP_DEBUG` is enabled in `.env`.

Otherwise, it will redirect to `/` (can be changed in `config/larascord.php`) and show the following alert:
![image](https://github.com/JakyeRU/Larascord/assets/36800842/59b85bc8-2a67-4b19-8caa-0ef1921d6a7b)

You can customize the aspect of the alert by editing `welcome.blade.php`.

**Reply by thephilluk** [reply]
Got it, thank you! Didn't even think about the `APP_DEBUG` flag.

Thanks for the fast reply
